setup.py

```diff
@@ -31,12 +31,9 @@
     + ';"arm" not in platform_machine and "aarch" not in platform_machine',
     "PyQtChart==5.13.1"
     + ';"arm" not in platform_machine and "aarch" not in platform_machine',
-    #
-    # FIXME: Maybe should be in a mode?
-    # qtconsole, pyserial
-    #
     "qtconsole==4.7.4",
     "pyserial==3.4",
+    "nudatus>=0.0.3",
```

Comment:
May or may not be relevant: 0.0.5 is the first to provide a wheel on PyPI

Reply:
Since this was the original pinned version when the upstream Debian package was created I thought it'd be best to leave it the same in case it is the only version available over there.